Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:13.12.0-alpine as build
+ENV NODE_ENV=prod
+WORKDIR /app/build
+COPY . .
+RUN npm install --prod
+RUN npm run build
+
+expose 80
```

Comment:
@eagunuworld  I think you want to expose it within the nginx step, also inconsistent case, i.e it should be upper case